### DISCUSSION TITLE
Grid squares not clickable to see QSOs

### DISCUSSION THREAD

**Discussion by M5JFS** [magicbug/Cloudlog#1089] [Ideas / Feature Requests]
Not sure if this is intended by design, but I have noticed that when I have sat QSOs that are multiple grids (line or corner), the squares show red or green in the grid squares analytics correctly, but the square is only clickable to see the QSOs if it was given in a single grid QSO, and in this case the multi-grid QSOs aren't shown in the list of QSOs.  If the square has only been part of a multi-grid QSO, the square is not clickable at all.

Also, the red dots don't appear to be shown on the map on the logbook homepage for multi-grid QSOs.

**Comment by magicbug** [comment]
Its by the design at the moment, I do need to go and look at that code however.
